tests/lean/change_tac.lean

```diff
@@ -45,3 +45,14 @@ example (h : 1 + 2 = 1 + 2 + 1) : 1 + 2 = 3 :=
 begin
   change 1 + 2 with 4 at *, -- ERROR
 end
+
+example : true :=
```

Comment:
Does this need a corresponding file with the expected output? I'm not sure what the test is right now, it should pass with any output from trace, right?

Reply:
There's already a file with the expected output, it just needs to be updated.

@cipher1024 You can run the tests locally on your machine by executing `ctest` in the build directory.  (Or `ctest -R leantest` if you only want to run the `tests/lean` tests.)  You can also run a single test using the `test_single.sh` script in the `tests/lean` directory.

Reply:
Sorry, I forgot it wasn't it my normal `make` command for Lean 3. The fix is coming